_layouts/sponsor_show.html

```diff
@@ -1,11 +1,19 @@
 ---
 layout: default
----
-
+---                
 <div id="new-sponsor-container">
-<a name="new"></a>
-<h3>Join Our Community of Sponsors</h3>
-<div>The Accidental Shakespeare Company would not exist without the generous support of our sponsors.  If you value our work and community presence, please consider becoming a sponsor.</div>
-<a id="new-sponsor-button" href="mailto:accshakes@gmail.com?subject=New Sponsor Inquiry">Become a sponsor</a>
+    <a name="new"></a>
+    <h2>Join Our Community of Sponsors</h2>
+    <div>The Accidental Shakespeare Company would not exist without the generous support of our sponsors.  If you value our work and community presence, please consider becoming a sponsor.</div>
+    <a id="new-sponsor-button" href="mailto:accshakes@gmail.com?subject=New Sponsor Inquiry">Become a sponsor</a>
```

Comment:
These links break on my browser (current version of Chrome) so we will need a fallback option for people who can't open mailto links.